### ISSUE TITLE
redirect to your repo

### ISSUE THREAD

**Issue by jtarchie** [telia-oss/github-pr-resource#34]
Hi!

In short...

Is alright to redirect people to you repo from mine?

I'd like to archive the github-pullrequest-resource in favor of yours. As it used the GraphQL, which solves a lot of issues.

We are fully feature complete, but I recognize that my resource started trying to do too much.

I'd also request that we update the concourse docs to reference your implementation rather than mine.

Cheers,

JT

**Comment by itsdalmo** [comment]
No problem at all, we are just happy to contribute what we can back to the Concourse community 🙌

Given that this resource is not a drop-in replacement for `github-pullrequest-resource` (e.g. parameters differ), perhaps we should provide a migration document to ease the transition?

**Comment by crstamps2** [comment]
I fully support this too. Its one of the main reasons I haven't really added in new features to the `github-pullrequest-resource` or taken the time to make the necessary tweaks to that resources CI and honestly even have started using this resource more. Using GraphQL is the better way to go. 

FWIW, a few colleagues and I at my office have talked about the problems that the original PR resource has, and what to do about them. I have some learnings (of personal growth on this) that it's hard to become the core maintainer of a project on its way to archival, because I didn't fully understand some of the decisions that JT made early on. 

Bottom line, I agree with JT on this. @jtarchie if you want me to, I can make the necessary changes on the original resource with deprecation warnings, etc.

**Comment by jtarchie** [comment]
@crstamps2, make it happen! Happy to help in any way I can, too.

There are no worries/concerns about ego, emotional attachment, etc. Our resource solved a problem at one point, but it no longer solves that issue effectively.

Better things are better. 👍 If something solves a problem better, use that thing.

🎉

**Comment by jtarchie** [comment]
Sorry, I forgot to update this link.

Once he merge of the docs happened on concourse-ci.org, we waited a month, and then closed down the resource.
